src/Aquifer.Jobs/SendNewContentEmails.cs

```diff
@@ -20,7 +20,7 @@ public class SendNewContentEmails(
     TelemetryClient telemetryClient)
 {
     [Function(nameof(SendNewContentEmails))]
-    public async Task Run([TimerTrigger("%MarketingEmail:CronSchedule:NewContent%")] TimerInfo timerInfo, CancellationToken ct)
+    public async Task Run([TimerTrigger("%MarketingEmail:CronSchedule:NewContent%")] TimerInfo _, CancellationToken ct)
```

Comment:
This broke Azure Functions bindings.  Fix here: https://github.com/BiblioNexusStudio/aquifer-server/pull/501